frontend.bat

```diff
@@ -523,14 +523,18 @@ for /F "tokens=1,2 delims=." %%a in ("!vsver!") do (
     rem https://github.com/3F/hMSBuild/issues/3
     set vsver=%%~a.0
 )
+REM Workaround for VS2019, may use 'vswhere -find' instead
+if !vsver! geq 16.0 set vsver=Current
```

Comment:
This could have used a vsver_major and made a numerical compare instead (reasonable safe this way).

Reply:
I agree, but `.0` is redundant, just `geq 16` will be better

Reply:
will change